pgrx-bindgen/src/build.rs

```diff
@@ -840,6 +852,31 @@ fn add_blocklists(bind: bindgen::Builder) -> bindgen::Builder {
         .blocklist_function("(?:sigstack|sigreturn|siggetmask|gets|vfork|te?mpnam(?:_r)?|mktemp)")
         // Missing on some systems, despite being in their headers.
         .blocklist_function("inet_net_pton.*")
+        // To make it work without `cshim`
```

Comment:
This is because this is the list we've already ported, I suppose?

Reply:
Yes. It avoids breaking changes.